.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 *dynamodb-local
+.idea/
```

Comment:
This should be in your personal .gitignore instead: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c